components/eamxx/src/physics/p3/eamxx_p3_process_interface.cpp

```diff
@@ -204,6 +205,12 @@ void P3Microphysics::initialize_impl (const RunType /* run_type */)
 {
   // Gather runtime options
   runtime_options.max_total_ni = m_params.get<double>("max_total_ni");
+
+  //////////// setting P3 constants in a struct
```

Comment:
These comment decorations are a little excessive, but I'm not strongly opposed.

Reply:
fixed, thanks